openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/database-schema-page.less

```diff
@@ -10,6 +10,8 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-
-export const activeLink = '#000000';
-export const normalLink = '#37352f';
+.database-schema-page-tabs {
+  .ant-tabs-nav {
+    margin: 0 !important;
+  }
+}
```

Comment:
Can we avoid this

Reply:
No we need this. Antd internally applied margin